Fabric.Identity.API/scripts/Install-Identity-Discovery-IdPSS.ps1

```diff
@@ -0,0 +1,62 @@
+﻿#Requires -RunAsAdministrator
+#Requires -Version 5.1
+#Requires -Modules PowerShellGet, PackageManagement
+
+param(
+    [PSCredential] $credential,
+    [ValidateScript({
+        if (!(Test-Path $_)) {
+            throw "Path $_ does not exist. Please enter valid path to the install.config."
+        }
+        if (!(Test-Path $_ -PathType Leaf)) {
+            throw "Path $_ is not a file. Please enter a valid path to the install.config."
+        }
+        return $true
+    })] 
+    [string] $installConfigPath = "$PSScriptRoot\install.config", 
+    [switch] $noDiscoveryService, 
+    [switch] $quiet
+)
+
+Test-MeetsMinimumRequiredPowerShellVerion -majorVersion 5
+
+if(!(Test-IsRunAsAdministrator))
+{
+    Write-DosMessage -Level "Error" -Message "You must run this script as an administrator. Halting configuration."
+    throw
+}
+
+# Import Fabric Install Utilities
+Import-Module -Name .\Install-Identity-Utilities.psm1 -Force
+
+$fabricInstallUtilities = ".\Fabric-Install-Utilities.psm1"
+if (!(Test-Path $fabricInstallUtilities -PathType Leaf)) {
+    Write-DosMessage -Level "Warning" -Message "Could not find fabric install utilities. Manually downloading and installing"
+    Get-WebRequestDownload -Uri https://raw.githubusercontent.com/HealthCatalyst/InstallScripts/master/common/Fabric-Install-Utilities.psm1 -NoCache -OutFile $fabricInstallUtilities
+}
+Import-Module -Name $fabricInstallUtilities -Force
+
+# common Settings
+$configStore = @{Type = "File"; Format = "XML"; Path = "$installConfigPath"}
+
+# Identity Settings
+$discoveryScope = "discovery"
+$discoveryConfig = Get-DosConfigValues -ConfigStore $configStore -Scope $discoveryScope
+$enableOAuth = [string]::IsNullOrEmpty($discoveryConfig.enableOAuth) -ne $true -and $discoveryConfig.enableOAuth -eq "true"
+
+if(!$enableOAuth){
+    # If AAD is not enabled, install discovery first.
+    .\Install-Discovery.ps1 -credential $credential -configStore $configStore -quiet
+}
+
+# Call the Identity powershell script
+.\Install-Identity.ps1 -credential $credential -configStore $configStore -noDiscoveryService:$noDiscoveryService -quiet
```

Comment:
Identity/IdPSS are not using a config store, they're using the path to the install.config instead

Reply:
So the script had it like this already.  I am pretty sure it uses the config store if it is presented.  This case is when Identity is installing.  If seperately, then the scenario you talk about is right.